spec/takeaway_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'takeaway'
```

Comment:
Really good unit tests both here and in your other specs. Nice lay out and easy to follow what's going on.